java/urllink/src/main/resources/test/application.properties

```diff
@@ -0,0 +1,80 @@
+#data source pool
+spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+spring.datasource.url=jdbc:mysql://10.0.0.95:3306/test?useAffectedRows=true&useUnicode=true&characterEncoding=utf-8&useSSL=false&serverTimezone=GMT%2B8&zeroDateTimeBehavior=convertToNull
+spring.datasource.username=ichbigdata
+spring.datasource.password=bigdata
+spring.datasource.initialSize=5
+spring.datasource.maxActive=50
+spring.datasource.maxIdle=20
+spring.datasource.minIdle=5
+spring.datasource.maxWait=60000
+spring.datasource.removeAbandonedTimeout=180
+spring.datasource.removeAbandoned=true
+spring.datasource.time-between-eviction-runs-millis=30000
+spring.datasource.test-on-borrow=false
+spring.datasource.test-while-idle=true
+spring.datasource.jdbcInterceptors=org.apache.tomcat.jdbc.pool.interceptor.SlowQueryReport(logFailed=true)
+
+
+
+#redis
+spring.redis.database=0
+spring.redis.host=10.0.0.95
+spring.redis.port=6379
+spring.redis.pool.min-idle=2
+spring.redis.pool.max-idle=6
+spring.redis.pool.max-active=8
+#second
+spring.redis.pool.max-wait=3000
+#second
+spring.redis.timeout=6000
+#minutes
+spring.redis.sesson.timeout=30
+
+#activemq
+spring.activemq.brokerUrl=tcp://10.0.0.217:61616?wireFormat.maxInactivityDuration=0
+spring.activemq.user=admin
+spring.activemq.password=A140F1CA244A4982975D810A60900DB2

```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 21854bf1-0856-452b-8ff3-62d29613354c -->